setup.cfg

```diff
@@ -46,3 +46,15 @@ max_line_length = 100
 
 [yapf]
 column_limit = 100
+
+[isort]
+line_length = 99
+not_skip = __init__.py
+sections=FUTURE,STDLIB,THIRDPARTY,ASTROPY,FIRSTPARTY,LOCALFOLDER
```

Comment:
I was gonna say we don't need `FUTURE` but I saw that in 3.7 that they added stuff under that header.

Reply:
yeah it's not specific to 2->3 although it's unlikely we will use them.